### DISCUSSION TITLE
[ERR_UNKNOWN_FILE_EXTENSION]: Unknown file extension ".ts"

### DISCUSSION THREAD

**Discussion by zhangfisher** [TypeStrong/ts-node#2144] [General]
```
[ERR_UNKNOWN_FILE_EXTENSION]: Unknown file extension ".ts"
```

The above errors repeatedly occur in different versions of 'ts node', but there has been no effective solution.

**Comment by zurmokeeper** [comment]
@zhangfisher How can you, downgrade the version of node.js to v18.18.x , after that there will be some problems using the version.
